waterlog/src/App.scss

```diff
@@ -1 +1,3 @@
-@import "./../node_modules/bootstrap/scss/bootstrap.scss";
\ No newline at end of file
+body {
```

Comment:
Same purpose as the inline style on the body tag.
Apply the DRY principle in when working with CSS. Don't repeat yourself.
You may remove the inline style and keep this one.

Reply:
removed inline styles, and used global variables